objcts.js

```diff
@@ -0,0 +1,9 @@
+for(i=1; i<=15;i++){
+let k = i;
```

Comment:
We don't need to create a new variable here, Just using `i` would be ok i.e If you remove this line, the next line would look like
```js
let j = i * i;
```